### PR TITLE
Dashboard/tooltips

### PR DESCRIPTION
**Pull request by tristanhcole**
**Describe the Pull Request**
Adds explainer tooltips to the top-level metrics and filter/group by. Feedback on definitions appreciated!

Screen Shot 2020-08-14 at 4 36 46 pm

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes etc.
- [x] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

**Comment by tristanhcole** [comment]
[Tooltips on to help clarify things](https://trello.com/c/FRkXCXvL/523-tooltips-on-to-help-clarify-things)

File: app/client/components/dashboard/card/CustomTabs.jsx [hunk 1]
```diff
@@ -96,8 +100,3 @@ export default class CustomTabs extends React.Component {
     );
   }
 }
-//
-// Tabs.PropTypes = {
-//   possibleTimeseries: PropTypes.arrayOf(PropTypes.string),
```
Comment (michielderoos) [inline app/client/components/dashboard/card/CustomTabs.jsx:101]: Should these be deleted?

Comment (tristanhcole) [reply app/client/components/dashboard/card/CustomTabs.jsx:101]: I think so, they are not used currently

**Comment by tristanhcole** [comment]
> All the uses of "whom" should be replaced with "who" (the user is the subject of the sentence - "whom" is used to refer to the object of a verb or preposition like "Sally is transacting with whom?")
> 
> Also should we review the word "transacted" and make sure we're consistent (relative to "transfer etc")?

fixed in latest commit

**Comment by tristanhcole** [comment]
> I think this needs an affordance like a "?" or "i" icon so that people know where there are a tooltips... thoughts?

I added it and thought it didn't look too good tbh. 

If you think necessary, happy to add.

**Comment by enjeyw** [comment]
> > I think this needs an affordance like a "?" or "i" icon so that people know where there are a tooltips... thoughts?
> 
> I added it and thought it didn't look too good tbh.
> 
> If you think necessary, happy to add.

Lemme throw an icon in on my side and stare at it for a second to see what i think

**Comment by enjeyw** [comment]
> > I think this needs an affordance like a "?" or "i" icon so that people know where there are a tooltips... thoughts?
> 
> I added it and thought it didn't look too good tbh.
> 
> If you think necessary, happy to add.

I do generally agree that having "?"s littered all over the page is ugly, but tooltips aren't useful unless people actually mouse over them lol

**Comment by enjeyw** [comment]
@tristanhcole I don't love the appearance of tooltip icons, but unfortunately I think they significantly increase usability. They're not bad if we use the filled variant and decrease the opacity (obvs we'd componentise if we implement)

:
```
<Tooltip title={tsPrompt}>
                      {toTitleCase(replaceUnderscores(tsLabel))}
                      <QuestionCircleFilled style={{marginLeft: 4, opacity: 0.4}}/>
</Tooltip>
```

Screen Shot 2020-08-17 at 10 11 01 pm

**Comment by tristanhcole** [comment]
> @tristanhcole I don't love the appearance of tooltip icons, but unfortunately I think they significantly increase usability. They're not bad if we use the filled variant and decrease the opacity (obvs we'd componentise if we implement)
> 
> :
> 
> ```
> <Tooltip title={tsPrompt}>
>                       {toTitleCase(replaceUnderscores(tsLabel))}
>                       <QuestionCircleFilled style={{marginLeft: 4, opacity: 0.4}}/>
> </Tooltip>
> ```
> 
> Screen Shot 2020-08-17 at 10 11 01 pm

Ok - implemented as a tooltip wrapper component in latest commit 😄

**Review by enjeyw — APPROVED** [review]
LGTM!

**Pull request merged**